website/docs/users-sources/user/user_basic_operations.md

```diff
@@ -120,3 +120,18 @@ You may instead deactivate the account to preserve identity data.
 2. Review the changes and click **Delete**.
 
 The user list refreshes and no longer displays the removed users.
+
+## Impersonate a user
+
+With authentik, an Admin can impersonate a user, meaning that the Admin temporarily assumes the identity of the user.
+
+1. In the Admin interface, navigate to **Directory > Users** to display all users.
+2. Click the name of the user to display the full User details page.
+3. On the Overview tab, beneath **User Details**, in the **Actions** area, click **Impersonate**.
+4. At the prompt, provide a reason why you are impersonating this user, and then click **Submit**.
+
+:::info
+An Admin can globally enable or disable impersonation in the [System Settings](../../sys-mgmt/settings.md#impersonation). By default, this option is set to true, meaning all users can be personified.
```

Comment:
Not sure `personified` is the correct word here

Reply:
Ah, yeah, we need to use the word "impersonated". Thanks for catching that!